components/popup/src/vwc-popup-base.ts

```diff
@@ -134,6 +134,7 @@ export class VWCPopupBase extends LitElement {
     		this.anchorEl = this.getAnchorById();
     	}
     	if (this.anchorEl && this.popupEl) {
+    		this.cleanup?.();
```

Comment:
what does this clean up?

Reply:
It makes sure that if we replaced the anchor element, it will not keep the old listener.